### ISSUE TITLE
Mismatch between displayed code and copied code

### ISSUE THREAD

**Issue by edhenderson** [themesberg/flowbite#325]
When I click copy on https://flowbite.com/docs/components/carousel/#indicators, I get:

```
<div id="indicators-carousel" class="relative" data-carousel="static">
    <!-- Carousel wrapper -->
    <div class="relative h-56 overflow-hidden rounded-lg md:h-96">
         <!-- Item 1 -->
        <div class="duration-700 ease-in-out absolute inset-0 transition-all transform translate-x-0 z-20" data-carousel-item="active">
            <img src="/docs/images/carousel/carousel-1.svg" class="absolute block w-full -translate-x-1/2 -translate-y-1/2 top-1/2 left-1/2" alt="...">
        </div>
:
```

But the code on the page is:

```
<div id="indicators-carousel" class="relative" data-carousel="static">
    <!-- Carousel wrapper -->
    <div class="relative h-56 overflow-hidden rounded-lg md:h-96">
         <!-- Item 1 -->
        <div class="hidden duration-700 ease-in-out" data-carousel-item="active">
            <img src="/docs/images/carousel/carousel-1.svg" class="absolute block w-full -translate-x-1/2 -translate-y-1/2 top-1/2 left-1/2" alt="...">
        </div>
```

As you can see the classes for _Item 1_ are different. Is this deliberate?

**Comment by zoltanszogyenyi** [comment]
Hello @edhenderson,

I'll have a look at this today. Thanks!

**Comment by zoltanszogyenyi** [comment]
Fixed for the `v1.5.4` update via the https://github.com/themesberg/flowbite/commit/c699f09850cd0cca2878c333eaf2462afa7bfaff commit.
